src/client.js

```diff
@@ -113,6 +113,63 @@ const successMessages = [
 ];
 
 
+function sendVoiceLine (messageType) {
```

Comment:
This looks like a bad fork conflict resolution, as it duplicates existing code.

Reply:
yep, I'll fix it